File: JavaScript/resistor_color_duo.js
const COLORS = ["black","brown","red","orange","yellow","green","blue","violet","grey","white"];

const decodedValue = (colorArray) => {
    let a = COLORS.indexOf(colorArray[0]);
    let b = COLORS.indexOf(colorArray[1]);
    
    return Number(String(a) + String(b));  
};

console.log(decodedValue(['blue', 'grey'])); // 68
console.log(decodedValue(['yellow', 'violet'])); // 47

File: C#/Resistor Color Duo/ResistorColorDuo.cs
using System;
using System.Linq;

public static class ResistorColorDuo
{
    private static readonly string[] _colors = ["black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"];

    public static int Value(string[] colors)
    {
        int a = Array.IndexOf(_colors, colors[0]);
        int b = Array.IndexOf(_colors, colors[1]);

        return int.Parse($"{a}{b}");
    }
}